libwget/html_url.c

```diff
@@ -66,8 +66,8 @@ static const char maybe[256] = {
 	['u'] = 1,
 };
 static const char attrs[][12] = {
-	"action", "archive",
-	"background",
+	"action", "archive",						// Add new entry in sorted position
+	"background",								// as binary search is performed on attrs[][]
 	"code", "codebase", "cite", "classid",
```

Comment:
Please do only use tabs at the beginning of a line. People have different tab settings (I use a width of 3), so they only make sense for indentation.